float64 left_sensor
float64 middle_sensor
float64 right_sensor
int8[3] error
